DOCKER.md

```diff
@@ -0,0 +1,87 @@
+## Docker
+
+In order to successfully test Qt.py against the different bindings of different versions, we use Travis-CI to run Docker containers based on [pre-built CentOS-based images](https://hub.docker.com/r/fredrikaverpil/qt.py/tags/).
+
+The Docker images follow the [VFX Reference Platform specifications](http://www.vfxplatform.com/) with some additionals, and are quite jam-packed.
+
+
+<br>
+<br>
+
+
+**Software built from source**
+
+* glibc<sup>1</sup>
+* gcc<sup>1</sup>
+* git<sup>2</sup>
+* cmake<sup>3</sup>
+* Qt4<sup>4</sup>
+* Qt5 + Autodesk-modifications<sup>1</sup>
+* Qt5 Creator<sup>1, 5</sup>
+* SIP<sup>4</sup>
+* PyQt4<sup>4</sup>
+* PySide<sup>4</sup>
+* PyQt5<sup>1</sup>
+* PySide2<sup>1</sup>
+
+<sup>1</sup> Per specification from VFX Platform  
+<sup>2</sup> Adds possibility to faster clone large repositories  
+<sup>3</sup> cmake 3.x required to build PySide2  
+<sup>4</sup> Required for Qt.py testing  
+<sup>5</sup> Required for `PySide2.QtUiTools`
+
+<br>
+<br>
+
+
+**Software versions**
+
+We use source code from around the time of [SIGGRAPH](http://www.siggraph.org) (usually July/August) each year. This is usually when the VFX Reference Platform is updated.
+
+In order to be able to re-build our images from a certain point in time, we cherry-pick commit SHAs or source archives rather than going for the always-latest version of software. We then set up new images as required and add them to our continous integration tests.
+
+There are non-strict rules (guidelines) for choosing software versions to build:
+
+- No commit SHA can be newer than the commit SHA from PySide2
+- No binding's commit SHA can be older than their respective Qt commit SHA
+
+Other noteworthy things:
+
+- PySide cannot be built with anything newer than Python 3.4.
+- SIP is used by both PyQt4 and PyQt5 and its version must therefore be chosen carefully.
+- PySide2 does not have a maintained `__version__` string.
+
+<br>
+<br>
+
+**Image tagging**
+
+A new image should be tagged like `repo/qt.py:YYYY`. For CY2017, this translates into `fredrikaverpil/qt.py:2017`.
+
+If there happens to be an update inbetween VFX Platform specifications, a revision version number could be added: `repo/qt.py:YYYY-update1`.
+
+<br>
+<br>
+
+
+**Dockerfile.vfxplatform2017**
```

Comment:
This is not the same table. Our table (in this PR) clearly shows from when/where each commit SHA was selected. This is nothing that the VFX Platform defines.

Down the line, we might experience issues with a certain commit and need to either change it or perhaps even create a `Dockerfile.vfxplatform2017-update1` since we want to be able to test for both cases. It would then be good to keep tabs somewhere on which commits were used.

Also, when adding CY2018, these table will make it easy to spot the exact differences between vfx platform revisions.

I'd like to keep this in. It helps when setting up the Dockerfiles. Since I commit changes to the table, I can go back in the git log to see exactly which commits didn't work and why.